packages/app/ui/components/SystemNotices.tsx

```diff
@@ -50,6 +50,7 @@ export function ContactBookPrompt(props: {
       const result = await perms.requestPermissions();
       if (result === 'granted') {
         await store.syncSystemContacts();
+        await store.syncContactDiscovery();
```

Comment:
can we make sure we always dismiss the notice if discovery fails? I'm guessing that'll have a higher error rate than `syncSystemContacts` which is all local.